pkg/cmd/server/server.go

```diff
@@ -364,3 +361,18 @@ func (c *completedServerConfig) Run(ctx context.Context) error {
 
 	return nil
 }
+
+var promOnce sync.Once
+
+// enableGRPCHistogram protects prom metric registration with a sync.Once.
+// Normally only one server will be constructed at a time, but the protection is
+// helpful for test scenarios where multiple servers may start in parallel.
+func enableGRPCHistogram() {
+	// EnableHandlingTimeHistogram is not thread safe and only needs to happen
+	// once
+	promOnce.Do(func() {
+		grpcprom.EnableHandlingTimeHistogram(grpcprom.WithHistogramBuckets(
+			[]float64{.006, .010, .018, .024, .032, .042, .056, .075, .100, .178, .316, .562, 1.000},
+		))
+	})
```

Comment:
I think it's more confusing to not just inline this, rather than putting it in a function.